test/test_dagbuilder.py

```diff
@@ -112,7 +112,8 @@ def test_should_generate_DAG_file_from_workflow_with_hourly_scheduling(self, get
     is_delete_operator_pod=True,
     retries=10,
     retry_delay=datetime.timedelta(seconds=20),
-    dag=dag)
+    dag=dag,
+    execution_timeout=None)
```

Comment:
nit: it is possible to add trailing `,` after last argument, which results into smaller diffs when you add/remove/reorder named arguments.